components/dropzone/stories/dropzone.stories.js

```diff
@@ -30,6 +30,7 @@ export default {
 			control: "boolean",
 			if: { arg: "isDragged", truthy: true },
 		},
+		reducedMotion: { table: { disable: true } },
```

Comment:
I added this to every non-animated component that I updated docs for as the [Contributing docs](https://opensource.adobe.com/spectrum-css/infieldbutton.html) say to add it for components that do not include any animation.